recipes/bandage_ng/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "BandageNG" %}
```

Comment:
I don't know if you've seen, linting complains about the name of the folder which is not matching the name in the recipe: https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=23915&view=logs&j=eb1ca712-fec5-5d93-deb0-40983fe61f2f&t=c9ebc308-e058-58af-a4f4-866fa7649dbb&l=27
you'll have to choose between bandage_ng and bandageng :)

Reply:
Thanks for that, just fixed :+1: